### PR TITLE
Enable etcd healthcheck

### PR DESCRIPTION
**Pull request by mshitrit**
As part of [ECOPROJECT-1073](https://issues.redhat.com//browse/ECOPROJECT-1073) adding etcd health check as part of control-plane node diagnostics.
Tested on local cluster.

### CODE REVIEW

File: pkg/controlplane/manager.go [hunk 1]
```diff
@@ -22,7 +22,8 @@ import (
 )
 
 const (
-	kubeletPort = "10250"
+	kubeletPort    = "10250"
+	etcdHealthPort = "9980"
```
Comment (razo7) [inline pkg/controlplane/manager.go:26]: Why did you choose this port? Is it a well known port for etcd health?

Comment (mshitrit) [reply pkg/controlplane/manager.go:26]: yes, it's taken from [here](https://github.com/openshift/cluster-etcd-operator/blob/master/bindata/etcd/pod.yaml#L205-L209)

File: main.go [hunk 2]
```diff
@@ -278,7 +278,7 @@ func initSelfNodeRemediationAgent(mgr manager.Manager) {
 
 	// but the reboot time needs be at least the time we know we need for determining a node issue and trigger the reboot!
 	// 1. time for determine node issue
-	minTimeToAssumeNodeRebooted := (apiCheckInterval + apiServerTimeout) * time.Duration(maxErrorThreshold) + maxTimeForNoPeersResponse
+	minTimeToAssumeNodeRebooted := (apiCheckInterval+apiServerTimeout)*time.Duration(maxErrorThreshold) + maxTimeForNoPeersResponse
```
Comment (slintes) [inline main.go:281]: why this change?

File: pkg/controlplane/manager.go [hunk 3]
```diff
@@ -82,13 +83,15 @@ func (manager *Manager) IsControlPlaneHealthy(workerPeerResponse peers.Response,
 }
 
 func (manager *Manager) isDiagnosticsPassed() bool {
+	manager.log.Info("Starting control-plane node diagnostics")
 	if manager.isEndpointAccessLost() {
 		return false
 	} else if !manager.isKubeletServiceRunning() {
```
Comment (slintes) [inline pkg/controlplane/manager.go:89]: no need for `else`

File: main.go (hunk 2 above)
Comment (mshitrit) [reply main.go:281]: Not something I've done manually, I assume it's due to update in the linter

File: pkg/controlplane/manager.go (hunk 3 above)
Comment (mshitrit) [reply pkg/controlplane/manager.go:89]: Thanks, I'll update in another PR :+1:
